ambry-replication/src/main/java/com/github/ambry/replication/ReplicationMetrics.java

```diff
@@ -259,6 +261,8 @@ public ReplicationMetrics(MetricRegistry registry, List<? extends ReplicaId> rep
         registry.histogram(MetricRegistry.name(ReplicaThread.class, "IntraColoOneCycleReplicationTime"));
     intraColoFastestSlowestReplicaPerCycleTimeDifference =
         registry.histogram(MetricRegistry.name(ReplicaThread.class, "IntraColoFastestSlowestReplicaPerCycleTimeDifference"));
+    intraColoRemoteReplicaGroupIdleTime =
+        registry.histogram(MetricRegistry.name(ReplicaThread.class, "IntraColoRemoteReplicaGroupIdleTime"));
```

Comment:
For brevity you can suffix the time unit in metric name like `IntraColoRemoteReplicaGroupIdleTimeMs`

Reply:
All the time units in across all metrics in ambry replication are in ms.
No other time metrics is having time unit suffix. This is an existing code smell. So can we keep it as such ?

Reply:
Please add Ms suffix because when we are reading code, knowing the suffix is immensely useful rather than digging into the code to understand the units. And when we are setting the configs, we will instantly know what value to set based on the suffix. Excluding suffixes in the past was wrong.

Reply:
On second thought, measure it in seconds. suffix = Secs.